### PR TITLE
add lightstep tracing propagators

### PR DESCRIPTION
**Pull request by vetinari**
this is #1652 redone with latest tracer

### CODE REVIEW

**Comment by szuecs** [comment]
@vetinari can you do a `go mod tidy` please?

**Comment by vetinari** [comment]
there's nothing to clean up for go mod tidy ... 
```
$ go mod tidy
$ git diff
$ 
```

File: tracing/tracers/lightstep/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# lightstep tracer
```
Comment (szuecs) [inline tracing/tracers/lightstep/README.md:1]: Can you please add a link from https://opensource.zalando.com/skipper/operation/operation/#opentracing (docs/operation/operation.md) to here? 
thanks!

Comment (vetinari) [reply tracing/tracers/lightstep/README.md:1]: linked to master where it will appear when it's merged

**Comment by szuecs** [comment]
:+1:

**Comment by vetinari** [comment]
:+1:

**Pull request merged**